tests/solana.rs

```diff
@@ -36,7 +34,7 @@ use std::{
 };
 use tiny_keccak::{Hasher, Keccak};
 
-mod borsh_encoding;
+pub mod borsh_encoding;
```

Comment:
Is this necessary? Works fine without `pub` here

Reply:
You're right. That should not be there.